src/main/java/seedu/address/logic/parser/todolistparsers/FindTaskParser.java

```diff
@@ -1,12 +1,94 @@
 package seedu.address.logic.parser.todolistparsers;
 
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PRIORITY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Stream;
+
 import seedu.address.logic.commands.todolistcommands.FindTaskCommand;
+import seedu.address.logic.parser.ArgumentMultimap;
+import seedu.address.logic.parser.ArgumentTokenizer;
 import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.logic.parser.Prefix;
 import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.task.Date;
+import seedu.address.model.task.FindTaskCriteria;
+import seedu.address.model.task.NameContainsKeywordsPredicate;
+import seedu.address.model.task.Priority;
+import seedu.address.model.task.TaskContainsTagsPredicate;
+import seedu.address.model.task.TaskMatchesDatePredicate;
+import seedu.address.model.task.TaskMatchesPriorityPredicate;
 
+/**
+ * Parses input arguments and creates a new FindTaskCommand object.
+ */
 public class FindTaskParser implements Parser<FindTaskCommand> {
-    @Override
-    public FindTaskCommand parse(String userInput) throws ParseException {
-        return null;
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the FindTaskCommand
+     * and returns a FindTaskCommand object for execution.
+     *
```

Comment:
Unnecessary blank space here as well. This applies to a few other javadoc comments in this pull request. Perhaps we should standardize this during a future team meeting.